config/samples/istio_v1beta1_meshgateway.yaml

```diff
@@ -25,6 +25,7 @@ spec:
     - port: 15443
       targetPort: 15443
       name: tls
+  additionalEnvVars: []
```

Comment:
We could add a commented out version of an example of env vars here (and to the istio CR as well) so that users would know what is the structure they need to provide for env vars.

Reply:
👍